models/web/scratch/snowplow_media_player_base_this_run.sql

```diff
@@ -50,6 +50,9 @@ with prep as (
     {%- elif target.type == 'bigquery' %}
     string_agg(cast(i.percent_progress as string), ',' order by i.percent_progress) as percent_progress_reached,
 
+    {%- elif target.type == 'databricks' %}
```

Comment:
Sorry, here I meant missing Spark.

Reply:
Thanks for highlighting them! Again this has been abstracted away in the other commit and spark is handled within the macro in the final version.